src/routes/Missions/description.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
+import Status from './status';
```

Comment:
- [ ] Kindly enable the eslint for good practice.
- [ ] Solution for removing that error you follow the following steps
1. run it in terminal `npm i -S prop-types'
2. then include `import PropTypes from 'prop-types';` into your js file
3. at the end of the code you can include this
 
` Rocket.propTypes = {
  rocket: PropTypes.shape({
    rocketName: PropTypes.string.isRequired,
    flickerImages: PropTypes.string.isRequired,
    description: PropTypes.string.isRequired,
  }).isRequired,
}; `

Reply:
Thanks @Mujeeb4582 
Let me implement these changes